### PR TITLE
refactor: Rebrand /pricing/pro

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Rebrand /pricing/pro based on [figma design](https://www.figma.com/design/vxxQijDub9nQrU3ZAYcXri/Pricing?node-id=0-1&node-type=canvas&t=zWIbN8ZHdZbmYrFX-0)
- Make tables more accessible

## QA

- Check that [the demo](https://ubuntu-com-14346.demos.haus/pricing/pro) matches the design
- Check that the page is accessible with keyboard and screen reader

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-12868

### CODE REVIEW

File: templates/pricing/pro.html [hunk 1]
```diff
@@ -60,21 +62,21 @@ <h2 class="u-no-margin--bottom">Ubuntu Pro pricing</h2>
           </thead>
           <tbody>
             <tr>
-              <td>
+              <th scope="row">
                 Desktop
```
Comment (mtruj013) [inline templates/pricing/pro.html:66]:
```suggestion
                Desktop or WSL
```

File: templates/pricing/pro.html [hunk 2]
```diff
@@ -561,40 +652,34 @@ <h3 class="p-heading--2">Ceph and Swift</h3>
...
             <tr>
-              <th width="35%">Feature</th>
-              <th class="u-align--right">Per VM</th>
-              <th class="u-align--right">Per host</th>
+              <th width="35%" scope="col">Feature</th>
+              <th class="u-align--right" scope="col">Per VM</th>
+              <th class="u-align--right" scope="col">Per host</th>
             </tr>
           </thead>
           <tbody>
```
Comment (mtruj013) [inline templates/pricing/pro.html:685]: It won't let me comment on the line directly, but on 691 on the asterisk we can add an aria label (aria-desribedby? I think that'd be the most appropriate attribute but could be wrong) to link it to the "On a VMware or OpenStack cluster..." text

**Review by mtruj013 — APPROVED** [review]
Thanks @petesfrench , LGTM with some super minor changes

File: templates/pricing/pro.html (hunk 2 above)
Comment (petesfrench) [reply templates/pricing/pro.html:685]: I was trying to get aria-describedby to work, as it seems the most appopriate but it turns out it has some different behaviours depending on whether it is used in a form or not. There is an interesting [stackoverflow post I was reading](https://stackoverflow.com/questions/56465819/accessibility-in-what-scenarios-would-aria-describedby-not-be-announced). In this scenario I had to use an aria-label and just explicitly add the footnote. Again, not sure if this is the 'correct' method, but it works

**Comment by eliman11** [comment]
There should be a colon after 'including' in that infra-only coverage section
Screenshot 2024-10-03 at 12 54 06

- Checked the copydoc and it seems like there shouldn't be a colon after 'system' in that first line on the table
- Add a space between 'repository' and '(Ubuntu 18.04 LTS onwards)' - this wasn't on the copydoc but I've updated it
Screenshot 2024-10-03 at 12 52 57

@mattea-turic

**Pull request merged**